tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+
+- set_fact: "es_major_version={{ es_version[0] }}.x"
+  when:
+      - not es_major_version
```

Comment:
This needs to be something like `es_major_version not defined`

Otherwise you get this error:

```
     TASK [elasticsearch : set_fact] ************************************************
       fatal: [localhost]: FAILED! => {"msg": "The conditional check 'not es_major_version' failed. The error was: error while evaluating conditional (not es_major_version): 'es_major_version' is undefined\n\nThe error appears to have been in '/tmp/kitchen/elasticsearch/tasks/main.yml': line 3, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- set_fact: \"es_major_version={{ es_version[0] }}.x\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes.  Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}
```